routing/transit_graph.cpp

```diff
@@ -124,6 +124,23 @@ void TransitGraph::Fill(vector<transit::Stop> const & stops, vector<transit::Edg
 
   StopToSegmentsMap stopToBack;
   StopToSegmentsMap stopToFront;
+  StopToSegmentsMap outgoing;
+  StopToSegmentsMap ingoing;
+
+  // It's important to add edges first to ensure fake segment id for particular edge is edge order in mwm.
```

Comment:
to add fake edges first?

Reply:
To add transit edges, fixed.
There are two parts of transit graph: transit::gates and transit::edges. Both transit::gates and transit::edges will have corresponding fake edges in transit graph. But we use transit::edges in cross-mwm and we need fake edges which correspond transit::edges to have stable ids.